.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: node_js
+sudo: false
```

Comment:
This is to use the container based infrastructure. The start time is a lot faster.